vmdb/app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -13,12 +13,12 @@ def textual_group_relationships
   end
```

Comment:
If this is not needed, it should be removed, I think?  You probably will have to remove from the view side as well.